build_tools/benchmarks/common/benchmark_driver_test.py

```diff
@@ -98,22 +99,20 @@ def tearDown(self) -> None:
   def test_add_previous_benchmarks_and_captures(self):
     driver = BenchmarkDriver(self.device_info, self.config,
                              self.benchmark_suite)
-    os.makedirs(os.path.join(self.tmp_dir.name, BENCHMARK_RESULTS_REL_PATH))
-    os.makedirs(os.path.join(self.tmp_dir.name, CAPTURES_REL_PATH))
-    benchmark_filename = os.path.join(
-        self.tmp_dir.name, BENCHMARK_RESULTS_REL_PATH,
+    os.makedirs(self.tmp_dir_path / BENCHMARK_RESULTS_REL_PATH)
+    os.makedirs(self.tmp_dir_path / CAPTURES_REL_PATH)
+    benchmark_filename = (
+        self.tmp_dir_path / BENCHMARK_RESULTS_REL_PATH /
         "MobileNetv2 [fp32,imagenet] (TFLite) big-core,full-inference with IREE-LLVM-CPU @ Pixel-4 (CPU-ARMv8.2-A).json"
     )
-    capture_filename = os.path.join(
-        self.tmp_dir.name, CAPTURES_REL_PATH,
+    benchmark_filename.write_text("")
+    capture_filename = (
+        self.tmp_dir_path / CAPTURES_REL_PATH /
         "MobileNetv2 [fp32,imagenet] (TFLite) big-core,full-inference with IREE-LLVM-CPU @ Pixel-4 (CPU-ARMv8.2-A).tracy"
     )
-    with open(os.path.join(benchmark_filename), "w") as f:
-      f.write("")
-    with open(os.path.join(capture_filename), "w") as f:
-      f.write("")
+    capture_filename.write_text("")
```

Comment:
Seems like you're just trying to create an empty file? In that case, https://docs.python.org/3.7/library/pathlib.html#pathlib.Path.touch

```suggestion
    capture_filename.touch()
```

Reply:
Done.